deployments/optimism.json

```
@@ -51,5 +51,8 @@
   "EmergencyPauseFacet": "0x6F2baA7cd5F156CA1B132F7FF11E0fa2aD775F61",
   "AcrossFacetV3": "0x6e00e0a7685Ca22c288d56D9E7924746B5043Ee7",
   "ReceiverAcrossV3": "0xB9CEc304899037E661F49DdFa7f64943b5920072",
+  "AcrossFacetPackedV3": "0x20F3FFf5A89e988c4109A6e496a839480B1B558f",
```

Biome (1.9.4)

[error] The key AcrossFacetPackedV3 was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)